### PR TITLE
Changes to invites

### PR DESCRIPTION
**Pull request by ahaadi**
Updates to Invites API. Add support for
1) Expiration datetime to be set on permission
2) Password to be set on permission
3) Examples of partial response.

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[51abc5e](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/51abc5ee81291bf5115610fd0ae1ba343e107827)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[51abc5e](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/51abc5ee81291bf5115610fd0ae1ba343e107827)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[51abc5e](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/51abc5ee81291bf5115610fd0ae1ba343e107827)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C21%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902210333248526-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6xhN1JrJVgqTbtlVBSYyGq2STqV519mMuFkhRRVzB2g%3D&st=2019-02-21T03%3A31%3A49Z&se=2019-03-24T03%3A36%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c5acb0c](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/c5acb0cc4b8d2cdbc6fc0feb761176bb268c10ee)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c5acb0c](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/c5acb0cc4b8d2cdbc6fc0feb761176bb268c10ee)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c5acb0c](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/c5acb0cc4b8d2cdbc6fc0feb761176bb268c10ee)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C28%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902282329060940-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ciy9Cw9rPPIS6ypRzv%2FDBSgn2X6BZNgDibf8jFiwGTI%3D&st=2019-02-28T23%3A27%3A29Z&se=2019-03-31T23%3A32%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3ccd08d](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/3ccd08d75355e3081390c41f69305893ffab31f3)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3ccd08d](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/3ccd08d75355e3081390c41f69305893ffab31f3)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3ccd08d](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/3ccd08d75355e3081390c41f69305893ffab31f3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903010127186821-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=idIS3EQpqQx%2FY66LjGiHQdUfCc%2F8lwzMMji%2B8wsanmY%3D&st=2019-03-01T01%3A25%3A55Z&se=2019-04-01T01%3A30%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ad98d26](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/ad98d2669033673eb1b9a0a00e8085217cfa1acc)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by ahaadi** [comment]
> Please add a changelog entry for the new properties.

Done. I added the new properties to concepts/changelog

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ad98d26](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/ad98d2669033673eb1b9a0a00e8085217cfa1acc)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ad98d26](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/ad98d2669033673eb1b9a0a00e8085217cfa1acc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903010131308948-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=67BG3sgnSHU4%2BOEc5LNIj697OqIzJwTrl48o12ITQFM%3D&st=2019-03-01T01%3A30%3A14Z&se=2019-04-01T01%3A35%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[86cfd09](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/86cfd0992fbffdfa243ab7f7b279669ad78e0960)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[86cfd09](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/86cfd0992fbffdfa243ab7f7b279669ad78e0960)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[86cfd09](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/86cfd0992fbffdfa243ab7f7b279669ad78e0960)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903041657128605-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qkjXyFX6IteUZ3WSFZ0YoiI%2FUMhP5Ic8C53AEcQbC%2Bs%3D&st=2019-03-04T16%3A55%3A52Z&se=2019-04-04T17%3A00%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3220f1c](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/3220f1caaa4a002be11de6b16ec3a49e0e6d8aae)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3220f1c](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/3220f1caaa4a002be11de6b16ec3a49e0e6d8aae)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

File: concepts/changelog.md [hunk 1]
```diff
@@ -513,6 +507,8 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 | Addition        | Beta        | Added the **location** property to the [itemActivity](/graph/api/resources/itemactivity?view=graph-rest-beta) complex type |
 | Addition        | v1.0        | Added the **preview** action on the [driveItem](/graph/api/resources/driveitem?view=graph-rest-1.0) entity |
 | Addition        | v1.0        | Added the [itemPreviewInfo](/graph/api/resources/itempreviewinfo?view=graph-rest-1.0) complex type |
+| Addition        | Beta        | Added the **expirationDateTime** property to the [driveItem_invite](/graph/api/resources/driveItem_invite?view=graph-rest-beta) DateTimeOffset type. |
```
Comment (Lauragra) [inline concepts/changelog.md]: You added these entries under September. The changes should be listed under March - unless you're documenting this retroactively?

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3220f1c](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/3220f1caaa4a002be11de6b16ec3a49e0e6d8aae)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903041702195732-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bkijOG2haQnaxilWl%2FoLdSTkEgD01QdUtZXAx6pfwVM%3D&st=2019-03-04T17%3A01%3A02Z&se=2019-04-04T17%3A06%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[92c531b](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/92c531baafa88df51d63babaaf07867ab7f1c328)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[92c531b](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/92c531baafa88df51d63babaaf07867ab7f1c328)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[92c531b](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/92c531baafa88df51d63babaaf07867ab7f1c328)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903041704427140-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RQ5u3VDnu%2BZT3uqkE69rAqrS%2FBAOQqsT4GUiQVGUsP4%3D&st=2019-03-04T17%3A05%3A02Z&se=2019-04-04T17%3A10%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9eeb4ac](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/9eeb4acc64eb1de9a50982099a0d22185eec60d1)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9eeb4ac](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/9eeb4acc64eb1de9a50982099a0d22185eec60d1)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9eeb4ac](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/9eeb4acc64eb1de9a50982099a0d22185eec60d1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903041910006443-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=psfLl6tfu%2BSMSxFwe96s6MnhyuhPGPiKqbO%2BtM%2FVhQ0%3D&st=2019-03-04T19%3A08%3A38Z&se=2019-04-04T19%3A13%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9cc1162](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/9cc1162de01da3af457fbe899fe0ed64f7dfa84c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9cc1162](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/9cc1162de01da3af457fbe899fe0ed64f7dfa84c)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9cc1162](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/9cc1162de01da3af457fbe899fe0ed64f7dfa84c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903041926138963-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sqLD8%2Fn7WoyHmO4T0QSsZfwQbtG8LI1%2B97hojvfeHyI%3D&st=2019-03-04T19%3A24%3A58Z&se=2019-04-04T19%3A29%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[686c836](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/686c836de03e461737e3dfbe7b668446d25697d7)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[686c836](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/686c836de03e461737e3dfbe7b668446d25697d7)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[686c836](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/686c836de03e461737e3dfbe7b668446d25697d7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903041932417493-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=i%2F0P4PnMut6exX6uMpBQiTqWwaytjlItbvKUu6EOwOw%3D&st=2019-03-04T19%3A31%3A35Z&se=2019-04-04T19%3A36%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[406ad9d](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/406ad9d90b59fe2c4a6d7187802fc761b7f81b49)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

File: concepts/changelog.md (hunk 1 above)
Comment (ahaadi) [reply concepts/changelog.md]: Thanks, moved them to March 2019.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[406ad9d](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/406ad9d90b59fe2c4a6d7187802fc761b7f81b49)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[406ad9d](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/406ad9d90b59fe2c4a6d7187802fc761b7f81b49)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903041959035211-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3qLgpt1OolzwP8pH172eXuXq9DNzJoBN8wLTQzSKt0k%3D&st=2019-03-04T19%3A57%3A45Z&se=2019-04-04T20%3A02%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kevklam** [comment]
@Lauragra  This one is ready to push now, but I don't appear to have permissions to apply the ready-to-merge label.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a76be86](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/a76be8624d7876daaaa45c3419df47c9895af93f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C15%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903152256459028-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9sGEfbt0taonOyWzmm8x1t5NJSWQRw%2BJzhO1EXXSU6Y%3D&st=2019-03-15T22%3A55%3A24Z&se=2019-04-15T23%3A00%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[1c94232](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/1c94232ac82540986d33ad73d11e5747e5ba1603)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C15%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903152259283342-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5eYqM1R7Ri%2BPO3aF7PLeW89pob76LMCDEdALP%2BHrCKE%3D&st=2019-03-15T22%3A59%3A24Z&se=2019-04-15T23%3A04%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[354a35f](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/354a35fd0b980171d74047db717460df9410c05b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903181630131017-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2x6k5c1MpBmGnFmO9WbTTUuG6glUltz0k5cmBjwcqys%3D&st=2019-03-18T16%3A28%3A56Z&se=2019-04-18T16%3A33%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[354a35f](https://github.com/ahaadi/microsoft-graph-docs/pull/4023/commits/354a35fd0b980171d74047db717460df9410c05b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/driveitem-invite.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/api-reference/beta/api/driveitem-invite.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/ahaadi/microsoft-graph-docs/blob/user/ahaadi/UpdateInvitesAPI/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903181630341383-4023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6uGxyUrhZiOYO7IHBbT5bz%2BK9FEOum5fw%2BExisT9wug%3D&st=2019-03-18T16%3A32%3A40Z&se=2019-04-18T16%3A37%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**